cmd/device-worker/main.go

```diff
@@ -156,6 +157,14 @@ func main() {
 	}
 	configManager.RegisterObserver(systemMetricsWatcher)
 
+	dataTransferWatcher := metrics.NewDataTransferMetrics(metricsDaemon)
+	if err != nil {
```

Comment:
there is no `err` returned by the call to `NewDataTransferMetrics`

Reply:
Done